tests/h/services/profile_group_test.py

```diff
@@ -122,3 +204,23 @@ def authority_group_service(pyramid_config):
     service.public_groups.return_value = None
     pyramid_config.register_service(service, name='authority_group')
     return service
+
+
+@pytest.fixture
+def pyramid_request(pyramid_request):
+    pyramid_request.route_url = mock.Mock(return_value='/group/a')
+    return pyramid_request
```

Comment:
You probably don't need it here, but in case you haven't noticed it there's a pattern for creating a `routes` fixture that actually defines routes in the Pyramid config (where they'll automatically be picked up by the `pyramid_request` fixture and methods like `route_url()`). Here's an example:

https://github.com/hypothesis/h/blob/c14662e83850f712883aeed1a5b7c71eb46d6992/tests/h/views/activity_test.py#L923-L929

These routes fixtures are usually used in an `@pytest.mark.usefixtures('routes')` on the test class, since you usually just want the same route(s) to be defined for all tests of a given class.

Reply:
Ah! I had stolen this current pattern from `session_test`; I'll take a peek at the `routes` fixture. Thanks for the heads up!